README.md

```diff
@@ -74,22 +79,24 @@ tokens you have received.
    that you can add a public DNS record for `kubectl` requests to find the router,
    hence the control plane instance.
 
-   `CLUSTER_TYPE` enables you to deploy cluster specific resources during 
+   `CLUSTER_TYPE` enables you to deploy cluster specific resources during
    installation. Currently the only supported types are `kind` or `aks`. `eks`
    and `gke` will be supported in a future release.
```

Comment:
Could we update this line to add `gke` to the list with `kind` and `aks`? It is supported and confirmed to work as of this moment in time.

Reply:
added